src/GraphQL/Execution/ExecutionStrategy.cs

```diff
@@ -132,12 +131,12 @@ public static void SetArrayItemNodes(ExecutionContext context, ArrayExecutionNod
                             + $" Field: {parent.Name}, Type: {parent.FieldDefinition.ResolvedType}.");
 
                         error.AddLocation(parent.Field, context.Document);
-                        error.Path = path;
+                        error.Path = parent.Path.Append(index.ToString());
```

Comment:
GetStringIndex ?

Reply:
GetStringIndex is a private member in ExecutionNode.cs  -- and as this is just exception handling, I didn't think it mattered much.

Reply:
Yes, I understand this, just wanted to make sure.